docs/spl-docs/timechart-command.md

```diff
@@ -0,0 +1,105 @@
+# timechart
+
+## Description
+
+Creates a time series chart with corresponding table of statistics.
+
+A timechart is a statistical aggregation applied to a field to produce a chart, with time used as the X-axis. You can specify a split-by field, where each distinct value of the split-by field becomes a series in the chart. With the `limit` you can specify series filtering. If you set limit=0, no series filtering occurs.
+
+## Syntax
+
+The required syntax is in **bold**.
+
+timechart \
+[limit=\<chart-limit-opt>] \
+[\<bin-options>... ] \
+(**\<single-agg>** [BY **\<split-by-clause>**] )
+
+## Required Arguments
+
+When specifying `timechart` command arguments, `<single-agg>`  BY `<split-by-clause>` is required.
```

Comment:
Is line 20 correct? From line 16, it looks like either `<single-agg>` or `<single-agg> BY <split-by-clause>` is required

Reply:
I just rechecked the documentation. Actually what you said was right, I got confused because the documentation mentioned it like this: `When specifying timechart command arguments, either <single-agg> or <eval-expression> BY <split-by-clause> is required.`

But the `BY split by clause` is mandatory for `eval-expression` but for `<single-agg>` this is optional.